### PR TITLE
Fix createOrUpdate endpoint

### PR DESCRIPTION
**Pull request by gquadrati**
#### List of Changes
- Added _installationId_ as path parameter within the _createorUpdate_ endpoint
- Updated the swagger.yaml file accordingly
- added _test.postman_collection.json_ with some basic tests

#### Motivation and Context
Fix the _CreateOrUpdate_ endpoint accordingly to the [documentation](https://docs.microsoft.com/it-it/rest/api/notificationhubs/create-overwrite-installation)

#### How Has This Been Tested?
It has been tested by performing `newman run test.postman_collection.json --insecure` after `yarn start`

#### Types of changes

- [ ] Refactor
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Review by BurnedMarshal — COMMENTED** [review]
Could you add to the readme the procedure to start the test?

**Comment by gquadrati** [comment]
> Could you add to the readme the procedure to start the test?

Sure.. done in [20ffb60](https://github.com/pagopa/io-notification-hub-mock/pull/16/commits/20ffb60477e025595f1bba9985a84c16ba8b8aba)

File: README.md [hunk 1]
```diff
@@ -8,31 +8,42 @@ It logs the information of the incoming requests, validates their payload, and s
...
+| DB_SOURCE         | The path of the JSON file used to store the data | number  |
+| EMAIL_PASSWORD    | The password used to connect to the email server | string  |
+| EMAIL_USER        | The user to connect to the email server          | string  |
+| EMAIL_SMTP_HOST   | The hostname or IP address of the SMTP server    | string  |
+| EMAIL_SMTP_PORT   | The port of the SMTP server                      | int     |
+| EMAIL_SMTP_SECURE | If set to true, uses TLS in SMTP connection      | boolean |
+| EMAIL_SENDER      | The email address of the sender                  | string  |
+| EMAIL_RECIPIENT   | The email address of the recipient               | string  |
+
+## Postman Collection Tests
```
Comment (BurnedMarshal) [inline README.md:40]: Please add the information that it is required that the app must be running before the execution of this test suite.

**Review by BurnedMarshal — APPROVED** [review]
LGTM apart a small doc improvement

File: README.md (hunk 1 above)
Comment (BurnedMarshal) [reply README.md:40]: thanks ;)

**Pull request merged**